project/src/com/google/daggerquery/QueryPlugin.java

```diff
@@ -0,0 +1,15 @@
+package com.google.daggerquery;
```

Comment:
Heads up, all source code files need to have a special header that includes copyright and license information.
https://github.com/googleinterns/dagger-query#source-code-headers

Reply:
✅